src/test/regress/regress_gp.c

```diff
@@ -637,12 +673,18 @@ resGroupPalloc(PG_FUNCTION_ARGS)
 	int size;
 	int count;
 	int i;
+	static bool startupConsidered = false;
 
 	if (!IsResGroupEnabled())
 		PG_RETURN_INT32(0);
 
 	ResGroupGetMemInfo(&memLimit, &slotQuota, &sharedQuota);
-	size = ceilf(memLimit * ratio);
+	if (!startupConsidered)
+		size = ceilf(memLimit * ratio) - 12;
+		startupConsidered = true;
+	else {
```

Comment:
seriously?!

Reply:
The ticket is not assigned to you and is not purposed for review